<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Vote_Menu</class>
 <widget class="QDialog" name="Vote_Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>237</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vote Menu</string>
  </property>
  <widget class="QPushButton" name="Vote_Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Vote</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Exit_Button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="Vote_Menu_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Vote Menu&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
